core/station/api/src/system.rs

```diff
@@ -97,22 +150,60 @@ pub struct InitAssetInput {
     pub decimals: u32,
 }
 
+#[derive(CandidType, serde::Serialize, Deserialize, Clone, Debug)]
+pub enum InitialConfig {
+    WithAllDefaults {
+        /// The initial users to create.
+        users: Vec<UserInitInput>,
+        /// The initial admin quorum in the admin level approval rule.
+        admin_quorum: u16,
+        /// The initial operator quorum in the operator level approval rule.
+        operator_quorum: u16,
+    },
```

Comment:
Could we avoid this variant by making the assets and accounts optional in the subsequent variant `WithDefaultPolicies`?

Reply:
I see the point, but then I wonder if the groups and status should be optional as their defaults are not very intuitive either.

Reply:
Yes I would make those non-optional as well. The user type was added by @keplervital. @keplervital what do you think of making the groups and status non-optional?

Reply:
let's make the `user_status` non optional then, but for groups i think it's more problematic, because ideally when using those `WithDefaults` and `WithDefaultPolicies` variant we would add the default group_ids for admin and operator, so if we want to avoid any confusion then we might need to have yet another UserInput that that is used for those variants that instead of and array of groups would have a boolean field like `is_admin`